datacats/environment.py

```diff
@@ -57,31 +57,56 @@ def __init__(self, name, target, datadir, ckan_version=None, port=None,
         self.datadir = datadir
         self.extension_dir = extension_dir
         self.ckan_version = ckan_version
+        # This is the child that all commands will operate on.
+        self.child_name = child_name
+        self.childdir = join(datadir, child_name)
         self.port = int(port if port else self._choose_port())
         self.deploy_target = deploy_target
         self.site_url = site_url
         self.always_prod = always_prod
+        self.children = None
 
-    def save(self):
+    def _load_children(self):
         """
-        Save environment settings into environment directory
+        Loads child names for this env from the .datacats-environment file.
+        This sets the property 'children' of the Environment.
+        This method is safe to call as long as a valid .datacats-environment
+        exists (it doesn't need a 'datacats/children' directive).
```

Comment:
Let's move the children in `datadir + '/children'` and then get this list from a listdir() call instead. The .datacats-environment file's children won't make sense if this file is shared to another user or machine.

I fully admit there are other settings in .datacats-environment that also might not make sense when shared, but let's not make things worse. :-)